frontend/src/services/auth/sessionUtils.ts

```diff
@@ -3,15 +3,25 @@
 import { KeyObject } from "crypto";
 import { JWTPayload, jwtVerify, SignJWT } from "jose";
 import { clientTokenExpirationInterval } from "src/constants/auth";
+import { environment } from "src/constants/environments";
 
 import { cookies } from "next/headers";
 
 export const CLIENT_JWT_ENCRYPTION_ALGORITHM = "HS256";
 export const API_JWT_ENCRYPTION_ALGORITHM = "RS256";
 
 // returns a new date 15 minutes from time of function call
-export const newExpirationDate = () =>
-  new Date(Date.now() + clientTokenExpirationInterval);
+export const newExpirationDate = () => {
+  if (
+    environment.AUTH_EXPIRATION_TIME !== "0" &&
+    environment.ENVIRONMENT === "dev"
```

Comment:
looks like "ENVIRONMENT" in dev will come through as "dev". To limit this to local only, can we tie this to something that for sure will only ever be set in a local environment? `ENVIRONMENT === 'local'` could work but we would need to make sure to set that when running the local server. 

Not sure why we're defaulting that to 'dev', but probably best to change that, since that suggests a deployed environment.

I'd advocate for creating a new NPM script that adds ENVIRONMENT="local", and we can get in the habit of using that when we don't want to be logged out?